src/main/java/net/dv8tion/jda/core/entities/impl/ReceivedMessage.java

```diff
@@ -149,6 +149,66 @@ else if (reaction.isSelf())
         return channel.addReactionById(getIdLong(), unicode);
     }
 
+    @Override
+    public RestAction<Void> removeReaction(Emote emote)
+    {
+        Checks.notNull(emote, "Emote");
+
+        MessageReaction reaction = reactions.parallelStream()
+            .filter(r -> Objects.equals(r.getReactionEmote().getId(), emote.getId()))
```

Comment:
compare the native longs

Reply:
same question as above, since addReaction does the same thing

Reply:
same answer as above i guess.